### DISCUSSION TITLE
Unable to deploy contract on Mumbai Polygon (CALL_EXCEPTION)

### DISCUSSION THREAD

**Discussion by Mcnoble1** [reach-sh/reach-lang#1531] [Q&A] [answered]
I'm facing this error while deploying to polygon mumbai testnet. I set my gas limit to 5000000, and still the error persists on both mumbai and goerli testnet.  
Then I used providerEnv to specify the ETH_NODE_URI ( RPC URL from my metamask) in my mjs file locally and imported accounts from Mnemonic that are setup on Polygon but got the same error.

Attached are error images with REACH_DEBUG on.
<img width="844" alt="11" src="https://user-images.githubusercontent.com/40045755/203845317-80a4ffda-c0e6-4f2c-97a7-a72e5f135b2e.png">
<img width="850" alt="12" src="https://user-images.githubusercontent.com/40045755/203845321-1bf5831f-3f4b-4a0f-b76a-765e69926987.png">
<img width="842" alt="13" src="https://user-images.githubusercontent.com/40045755/203845324-7aafa1ea-188d-4413-82ff-69d2cd6c118d.png">
<img width="845" alt="14" src="https://user-images.githubusercontent.com/40045755/203845327-02c515a4-d201-4719-b151-4e66a5c9e6a6.png">

**Comment by nstanford5** [comment]
For reference, this error does not occur in devnet

**Comment by Mcnoble1** [comment, marked as answer]
I've been able to fix the error.
I was using the `loadstdlib()` twice in my code. 
`loadstdlib() and loadstdlib(process.env)`
